test/tools-test.cpp

```diff
@@ -57,17 +57,47 @@ TEST(CommonTools, decodeUrl)
 
 TEST(CommonTools, computeRelativePath)
 {
-  EXPECT_EQ("B", computeRelativePath("A", "B"));
+  EXPECT_EQ("A",  computeRelativePath("A" , "A" ));
+  EXPECT_EQ("..", computeRelativePath("A/", "A" ));
+  EXPECT_EQ("A/", computeRelativePath("A" , "A/"));
+  EXPECT_EQ("./", computeRelativePath("A/", "A/"));
+
+  EXPECT_EQ("B",     computeRelativePath("A" , "B" ));
+  EXPECT_EQ("../B",  computeRelativePath("A/", "B" ));
+  EXPECT_EQ("B/",    computeRelativePath("A" , "B/"));
+  EXPECT_EQ("../B/", computeRelativePath("A/", "B/"));
+
+  EXPECT_EQ("A/B",  computeRelativePath("A" , "A/B" ));
+  EXPECT_EQ("B",    computeRelativePath("A/", "A/B" ));
+  EXPECT_EQ("A/B/", computeRelativePath("A" , "A/B/"));
+  EXPECT_EQ("B/",   computeRelativePath("A/", "A/B/"));
+
+  EXPECT_EQ("..",     computeRelativePath("A/B",  "A" ));
+  EXPECT_EQ("../..",  computeRelativePath("A/B/", "A" ));
+  EXPECT_EQ("../",    computeRelativePath("A/B",  "A/"));
+  EXPECT_EQ("../../", computeRelativePath("A/B/", "A/"));
+
```

Comment:
Can you add some tests (the same ?) with leading "/"  ?

Reply:
Currently `computeRelativePath()` is used only with paths returned by `zim::Entry::getPath()` which don't contain a leading slash.

Reply:
The comment above `computeRelativePath()` also states that the input paths are expected to be relative paths:  https://github.com/openzim/zim-tools/blob/ff61a93e2219407f25d31494e722a9b98d35488b/src/tools.h#L165-L168